Update openapi-aot.md (#34341)
dotnet/AspNetCore.Docs: aspnetcore/release-notes/aspnetcore-9/includes/openapi-aot.md

### Microsoft.AspNetCore.OpenApi supports trimming and Native AOT

> The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT.

Author: ```suggestion
The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish and OpenAPI app with trimming and Native AOT:
```

> Trimming and Native AOT are now available for OpenAPI in ASP.NET Core. The following steps create and publish an OpenAPI app with trimming and Native AOT:

Reviewer: ```suggestion
OpenAPI in ASP.NET Core supports trimming and native AOT. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```

> The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish an OpenAPI app with trimming and Native AOT:

Author: ```suggestion
Trimming and Native AOT are now available for OpenAPI in ASP.NET Core. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```

> The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish and OpenAPI app with trimming and Native AOT:

Author: ```suggestion
The new built-in OpenAPI in ASP.NET Core supports trimming and Native AOT. The following steps create and publish an OpenAPI app with trimming and Native AOT:
```
